components/public-api-server/pkg/apiv1/workspace.go

```diff
@@ -324,7 +324,7 @@ func convertWorkspaceInfo(input *protocol.WorkspaceInfo) (*v1.Workspace, error)
 		Context: &v1.WorkspaceContext{
 			ContextUrl: input.Workspace.ContextURL,
 			Details: &v1.WorkspaceContext_Git_{Git: &v1.WorkspaceContext_Git{
-				NormalizedContextUrl: input.Workspace.ContextURL,
+				NormalizedContextUrl: input.Workspace.Context.NormalizedContextURL,
```

Comment:
Does it make sense to expose the raw url in public api in the first place? I thought it's more an internal thing on server side, if it's really needed to be exposed then it should be renamed to something else because it's not an url in the first place

Reply:
We discussed this during teaming and I agree: `ContextURL` seems to be more of a burden than a benefit to have. The normalized one should be the default

Reply:
The question is why it was modelled like that in first place and with whom we need to discuss it then?